state-machines/mem/src/mem_planner.rs

```diff
@@ -1,6 +1,14 @@
 use std::sync::Arc;
 
 use sm_common::{BusDeviceMetrics, ChunkId, Plan, Planner};
+#[cfg(feature = "debug_mem")]
+use crate::MemDebug;
+#[cfg(feature = "debug_mem")]
+use zisk_pil::MEM_ALIGN_AIR_IDS;
+
+#[cfg(feature = "debug_mem")]
+use crate::{MemAlignCheckPoint, MemHelpers};
```

Comment:
This crates are not used. Same with MEM_ALIGN_AIR_IDS. On the other hand, MEM_BYTES should be imported or otherwise will not compile with features debug_mem

Reply:
I fixed it, removing no used parts, and use MemHelpers to get addr from addr_w